.github/workflows/scanning.yaml

```diff
@@ -1,20 +1,10 @@
 name: 'Vulnerabilities scanning'
-
 on:
   push:
 
-env:
-  DEP_PATH: requirements.txt
-
 jobs:
   scan:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check out master
-        uses: actions/checkout@master
-      - name: Security vulnerabilities scan
-        continue-on-error: false
-        uses: aufdenpunkt/python-safety-check@master
-        with:
-          scan_requirements_file_only: 'true'
-          safety_args: '--policy-file .safety-policy.yaml'
+    name: Vulnerabilities scan
+    uses: th2-net/.github/.github/workflows/python-scan.yml@python-workflow
+    with:
+      safety-args: '-i 70612'
```

Comment:
@davitmamrikishvili some comment is required here. No one will understand what that means and why we ignore 70612

Reply:
done

Reply:
This not explains why we ignore it. 
`transitive dependency.` is not the reason why we ignore this.
Otherwise, we should ignore all transitive dependencies, right?

Reply:
Updated. is it okay?

Reply:
```
safety-args: '-i 70612' # ignore https://data.safetycli.com/v/70612/97c/. Jinja2 library is not used in our code directly.
```
why Jinja2 is in our scope for checking ?

Reply:
ok.

Reply:
@Nikita-Smirnov-Exactpro I can't answer that, but we could easily limit the scope to the packages defined in requirements.txt.

Reply:
```
Full paths:
th2-data-services-utils -> vaex -> vaex-jupyter -> bqplot -> ipyfleat-> branca -> Jinja2 [required: >=3, installed: 3.1.3]
```
Could we update version of Jinja2 instead of ignore problem? Update is more applicable because we will not bring problem dependency into client environment.